### PR TITLE
Upgraded build tools

### PR DESCRIPTION
**Pull request by icancode23**
upgraded the buld tools version to 25.2.0

### CODE REVIEW

**Comment by icancode23** [comment]
@NickHub4017

**Comment by NickHub4017** [comment]
Updating the build tool version to current high

File: kute-android-app/.idea/modules.xml [hunk 1]
```diff
@@ -3,6 +3,8 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
+      <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
+      <module fileurl="file://$PROJECT_DIR$/kute-android-app.iml" filepath="$PROJECT_DIR$/kute-android-app.iml" />
```
Comment (CloudyPadmal) [inline kute-android-app/.idea/modules.xml]: These are duplicate lines it seems!

File: kute-android-app/app/build.gradle [hunk 2]
```diff
@@ -1,8 +1,23 @@
 apply plugin: 'com.android.application'
 
+<<<<<<< HEAD
```
Comment (NickHub4017) [inline kute-android-app/app/build.gradle]: please check this file.. It has unresolved conflicts.

Comment (icancode23) [inline kute-android-app/app/build.gradle]: @NickHub4017 
yes sir it was a small confusion doing it within 10 to 15 minutes

**Comment by icancode23** [comment]
@NickHub4017 done sir

**Pull request merged**